qutebrowser/misc/miscwidgets.py

```diff
@@ -105,13 +105,6 @@ def set_prompt(self, text):
         self._validator.prompt = text
         self._promptlen = len(text)
 
-    def home(self, mark):
```

Comment:
This indeed isn't needed anymore - 15b8c553c8dd011a7a7ae6b3262840f079566bbc suggests it was added to fix `rl-unix-line-discard`, but that seems to work fine nowadays without it (even on Qt 5.7.1).